mlir/lib/Dialect/MIOpen/Transforms/ThreadwiseGemmLowering.cpp

```diff
@@ -163,6 +163,14 @@ struct XdlopsGemmV2RewritePattern : public OpConversionPattern<XdlopsGemmV2Op> {
     int64_t MPerWave = tuningParams.getMPerWave();
     int64_t NPerWave = tuningParams.getNPerWave();
 
+    // Workload of either MPerWave and NPerWave that are larger
+    // than wave size of 64 will be executed by repeats
+    // TODO: amend this for tuning parameter selection as well
+    int64_t MRepeats = (MPerWave > 64) ? (MPerWave / 64) : 1;
```

Comment:
Poking this comment, especially since I'm not liking these harcoded 64s

Though an immediate fix would be calling that number `waveSize` so we can keep an eye on it later.

Reply:
You are right about repeats, they should be gone since that was the point of reshaping from blockwisegemm. I'd have to keep M/N Per Xdlops computation just so that it is able to inquire on the xdlops code selection again.